deploy/helm/flightctl/templates/_helpers.tpl

```diff
@@ -1,13 +1,21 @@
 {{- define "flightctl.getBaseDomain" }}
   {{- if .Values.global.baseDomain }}
     {{- printf .Values.global.baseDomain }}
-  {{- else }}
-    {{- $openShiftBaseDomain := (lookup "config.openshift.io/v1" "DNS" "" "cluster").spec.baseDomain }}
-    {{- if .noNs }}
-      {{- printf "apps.%s" $openShiftBaseDomain }}
+  {{- else if eq .Values.global.target "acm" }}
+    {{- /* For ACM deployments on OpenShift, try to lookup the base domain */}}
```

Comment:
This shouldn't be specific to ACM, but apply to all deployments on OpenShift.

Reply:
fixed